README.md

```diff
@@ -202,6 +202,8 @@ Usage of oauth2_proxy:
   -display-htpasswd-form: display username / password login form if an htpasswd file is provided (default true)
   -email-domain value: authenticate emails with the specified domain (may be given multiple times). Use * to authenticate any email
   -flush-interval: period between flushing response buffers when streaming responses (default "1s")
+  -skip-jwt-bearer-tokens: will skip requests that have verified JWT bearer tokens
```

Comment:
Can we add `(default false)` at the end of this please (assuming that is the default we have opted for?)

Reply:
Added suggestion